lib/src/common/observer_controller.dart

```diff
@@ -501,34 +501,125 @@ mixin ObserverControllerForScroll on ObserverControllerForInfo {
     final targetChild = findCurrentFirstChild(obj);
     if (targetChild == null) return;
     final isHorizontal = obj.constraints.axis == Axis.horizontal;
-    var nextChild = obj.childAfter(targetChild);
-    nextChild ??= obj.childBefore(targetChild);
-    double separatorTotalHeight = 0;
-    if (nextChild != null && nextChild is! RenderIndexedSemantics) {
-      // It is separator
-      final nextChildPaintBounds = nextChild.paintBounds;
-      final nextChildSize = isHorizontal
-          ? nextChildPaintBounds.width
-          : nextChildPaintBounds.height;
-      separatorTotalHeight = index * nextChildSize;
-    }
     final childPaintBounds = targetChild.paintBounds;
-    final childSize =
+    double itemSeparatorHeight = 0;
+    int indexLineCount = index;
+    if (obj is RenderSliverList || obj is RenderSliverFixedExtentList) {
```

Comment:
This judgment will cause problems with invalidate the jump of the third-party package's ListView.

Reply:
ok，I will improve this piece